### DISCUSSION TITLE
How long can a cron job run?

### DISCUSSION THREAD

**Discussion by hussamkhatib** [vercel/community#4669] [Feature Question] [answered]
### Question

This [section in the docs](https://vercel.com/blog/cron-jobs#get-started-with-vercel-cron-jobs) has a hyperlink to what I am looking for, but I couldn't find what the title suggests "How long can a cron job run?'.

**Comment by amyegan** [comment, marked as answer]
Hi @hussamkhatib! Thanks for reporting that incorrect link. I'll get those fixed right away.  In the meantime, I think this is what you want:

> The duration limits for Cron jobs are identical to those of [Serverless](https://vercel.com/docs/functions/serverless-functions#limits) and [Edge](https://vercel.com/docs/functions/edge-functions/limitations) functions. See the [maxDuration](https://vercel.com/docs/functions/serverless-functions/runtimes#max-duration) documentation for more information.

https://vercel.com/docs/cron-jobs/manage-cron-jobs#cron-job-duration

**Reply by hussamkhatib** [reply]
Thanks, that's helpful.
